### PR TITLE
Add Azure image listing to workflow

### PR DESCRIPTION
**Pull request by FKolwa**
Example output: https://github.com/redhatcloudx/rhelocator/actions/runs/3240256203

### CODE REVIEW

File: .github/workflows/container_release.yaml [hunk 1]
```diff
@@ -40,9 +40,11 @@ jobs:
       - name: Install dependencies
         run: poetry install
 
+      - name: Create output folder
+        run: mkdir ./results >/dev/null 2>&1
```
Comment (miyunari) [inline .github/workflows/container_release.yaml:44]: Why do we need to create the results folder like this upfront? :)

Comment (FKolwa) [reply .github/workflows/container_release.yaml:44]: Tbh, just to make sure it's really there. We can create it once we start creating something in that sub-directory, but that would still require us to create the folder before starting to pipe. But honestly there are probably a million more efficient ways to do anything in bash :D

Comment (miyunari) [reply .github/workflows/container_release.yaml:44]:
```suggestion
        run: mkdir -p results
```

Would that be the same?

**Comment by major** [comment]
@FKolwa This looks good, but it would be better to use @miyunari's `mkdir -p` suggestion to avoid confusion.

File: .github/workflows/container_release.yaml (hunk 1 above)
Comment (FKolwa) [reply .github/workflows/container_release.yaml:44]: Jap it would ;) Let's do it our way and increase readability!

**Comment by FKolwa** [comment]
> @FKolwa This looks good, but it would be better to use @miyunari's `mkdir -p` suggestion to avoid confusion.

Agreed

**Pull request merged**